### DISCUSSION TITLE
ظاهر جدید Subscription

### DISCUSSION THREAD

**Discussion by xematin** [Gozargah/Marzban#593] [General]
سلام . آیا این آدرس https://github.com/MuhammadAshouri/marzban-templates جدیدترین ظاهر بخش Subscription است ؟ چون ظاهر خیلی متفاوتی رو دیدم که با کلیک روی دکمه وارد برنامه میشه و کانفیگ یا سابسکریپشن خودکار داخل برنامه Paste میشه! ممنون میشم راهنمایی کنید 3>

**Comment by x0sina** [comment]
سلام دوست عزیز این تمپلیت با تمپلیتی که شما میفرمایین دوتا تمپلیت متفاوت هستش، میتونید قابلیت کلیک و اضافه شدن در برنامه رو از این تمپلیت استفاده کنید
[Marzban-sub](https://github.com/x0sina/marzban-sub)
